.github/workflows/build_package.yml

```
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    branches: "**"
+jobs:
+  release:
+    name: Release on GitHub and PGXN
+    runs-on: ubuntu-latest
+    container: pgxn/pgxn-tools
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Bundle the Release
+        id: bundle
+        run: pgxn-bundle
+      - name: Release on PGXN
+        env:
+          MAIN_BRANCH: pgxn-citus
+          PGXN_USERNAME: ${{ secrets.PGXN_USER_NAME }}
+          PGXN_PASSWORD: ${{ secrets.PGXN_PASSWORD }}
+        run: |
+          CURRENT_BRANCH="${GITHUB_REF##*/}"
+          if [ "${CURRENT_BRANCH}" = "${MAIN_BRANCH}" ]
+          then
+            pgxn-release
+          else
+            echo "Distro will not be released since current branch '${CURRENT_BRANCH}' is different from main branch '${MAIN_BRANCH}'"
+          fi
```

```suggestion
        if: ${{ github.ref == ${MAIN_BRANCH} }}
        run: pgxn-release
```

I haven't checked if I need some more quotation marks there. Please double check if this makes sense

---

It will not work since github.ref is /refs/HEAD/{branch_name}.

---

I want to see the detailed message when it is not pushed as well